src/sage/functions/special.py

```diff
@@ -457,7 +457,7 @@ class EllipticE(BuiltinFunction):
         elliptic_e(z, 1)
         sage: # this is still wrong: must be abs(sin(z)) + 2*round(z/pi)
         sage: elliptic_e(z, 1).simplify()
-        2*round(z/pi) + sin(z)
+        2*round(z/pi) ... sin(...z)
```

Comment:
Really, it seems to me the previous answer is wrong, and the new answer is correct, even if it's not the nicest (note that `abs(sin(z)) + 2*round(z/pi)` is also incorrect). Let's just write something like
```
sage: elliptic_e(z, 1).simplify()                # not tested: bad answer with maxima < 5.47
2*round(z/pi) - sin(pi*round(z/pi) - z)
```
(is there a way to flag a test as "expected to fail" rather than "not test" ?)

This is an even better test which shows the new answer is ok:
```
sage: f(z) = elliptic_e(z, 1).simplify()
sage: all([abs(f(z).simplify()-elliptic_e(z,1))<1e-15
....:      for z in [RR.random_element(10) for _ in range(10)]])
True
```
you can see this passes with maxima 5.47 but fails with maxima 5.46.

Reply:
Could do something like:
```
sage: maxima.version() < '5.47' or all([
....:     abs(f(z).simplify()-elliptic_e(z,1))<1e-15
....:     for z in [RR.random_element(10) for _ in range(100)]])
True
```
but it's a bit ugly :shrug:   (ideal would be a way to label the test something  like `xfail when maxima.version() < '5.47'`.